### PR TITLE
C2C-32: Cleaned the echographie form

### PR DESCRIPTION
**Pull request by reagan-meant**
Need some fr locale proof reading

### CODE REVIEW

File: configuration/concepts/echographie.csv [hunk 1]
```diff
@@ -0,0 +1,48 @@
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+89e9cd9c-be64-491a-a96c-1201f4a74b0b,,CIEL:152305,Confirmed Pregnancy,Grossesse confirmée,Confirmed Pregnancy,Grossesse confirmée,,,Misc,Coded,,Yes; No,,,,,,,,,
+a7ad184c-b79a-476d-9ba2-ef980375da28,,CIEL:165387,Foetus Number,Numéro du fœtus,Foetus #,Numéro du fœtus,,,Misc,Numeric,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv]: This should be 'Number of foetuses/Nombre de fœtus' right?

File: configuration/concepts/echographie.csv [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+c1e61ae3-4de8-4fe1-9b7a-8c1ae56d1631,,,Number of Placenta,Nombre de placenta,Number of Placenta,Nombre de placenta,,,Misc,Numeric,,,,,,,,,,,
+7ce5a9fe-d9e0-427e-a080-ce8a9c673822,,CIEL:1440,Ultrasound Fetal heart rate,Rythme cardiaque fœtal,Fetal heart rate,Rythme cardiaque fœtal,,,Misc,Numeric,,,,FALSE,beats/min,120,160,40,250,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv]: We should use the already existing 'Fetal Heart Rate' concept.

File: configuration/concepts/echographie.csv [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+c4526510-3f10-11e4-adec-0800271c1b75,,CIEL:160091,Cephalic,Céphalique,Cephalic,Céphalique,,,Misc,N/A,,,,,,,,,,,
+a0dc5727-c33e-479b-b70e-fb0eb7e45853,,CIEL:146922,Breech,Siège,Breech,Siège,,,Misc,N/A,,,,,,,,,,,
+5bb2ea3b-a736-4533-bd5e-a1e9d835f721,,CIEL:112259,Transverse,Transversale,Transverse,Transversale,,,Misc,N/A,,,,,,,,,,,
+cbafba29-b62a-4675-910a-a6bb161de256,,,Variable,Variable,Variable,Variable,,,Misc,N/A,,,,,,,,,,,
+d47e2e11-355e-4c62-86a7-8ebe45fb4292,,CIEL:160090,Ultrasound Presentation,Présentation,Presentation,Présentation,,,Misc,Coded,,Cephalic;Breech;Transverse;Variable,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv]: Same here, let's use the already existing 'Presentation.Présentation'

File: configuration/concepts/echographie.csv [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+7fc52ed8-8397-4016-be5a-b4686ef8fa7e,,CIEL:114423,Ovarian Pregnancy,Grossesse ovarienne,Ovarian Pregnancy,Grossesse ovarienne,,,Diagnosis,N/A,,,,,,,,,,,
+a999e659-d882-425e-b3db-a2d039d15ac6,,,Other Ectopic Pregnancy,Autre grossesse extra-utérine,Other Ectopic Pregnancy,Autre grossesse extra-utérine,,,Diagnosis,N/A,,,,,,,,,,,
+b4b1d099-1932-4eda-bc72-7ba69b471b3a,,,Intrauterine Pregnancy,Grossesse intra-utérine,Intrauterine Pregnancy,Grossesse intra-utérine,,,Diagnosis,N/A,,,,,,,,,,,
+309fba9e-7936-42e6-a7b6-d05830f5a432,,,Location of pregnancy,Localisation de la grossesse,Location of pregnancy,Localisation de la grossesse,,,Misc,Coded,,Intrauterine Pregnancy; Abdominal Pregnancy; Tubal Pregnancy; Ovarian Pregnancy; Other Ectopic Pregnancy,,,,,,,,,
+05e696ee-9330-483d-8419-1c587a5dd84e,,,Cervical Length,Longueur cervicale,Cervical Length,Longueur cervicale,,,Misc,Numeric,,,,TRUE,cm,,,,,,
+6422a83f-acc6-4953-8239-f18a0b6d90a4,,,Ultrasound Estimated date of delivery,Date prévue d'accouchement,Estimated date of delivery,Date prévue d'accouchement,,,Misc,Date,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv]: Same here, let's use the already existing 'Estimated date of delivery'

File: configuration/concepts/echographie.csv [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+9e071d37-ee01-4785-a756-7e4059ffe736,,CIEL:1438,Ultrasound Gestational Age,Âge gestationnel par ultrasons,Gestational Age,L'âge gestationnel,,,ConvSet,N/A,,,Number of Weeks; Number of Days,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv]: And here too

File: configuration/concepts/echographie.csv [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+4d1f668f-f9b7-4e95-8436-53f37e8c643e,,,Other observations,Autres observations,Other observations,Autres observations,,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv:49]: This sounds like a generic concept of 'Other' (free text). Is there such a concept already existing?
This concept seems to be part of the "Obstetric ultrasound" set so we'll always be able to diférenciate it from other 'Other' used in various forms.

File: configuration/concepts/echographie.csv [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+6a228d2d-87ca-4d84-a774-ebc0e26b55c9,,,Identified complications,Complications identifiées,Identified complications?,Complications identifiées?,,,Misc,Coded,,Yes; No,,,,,,,,,
+1fac1715-4e6e-4f87-ad75-d4ecf8bbdbe8,,CIEL:1576,Complications,Complications,"If yes, which ones","Si oui, lesquelles",,,Misc,Text,,,,,,,,,,,
+a3cfb4f1-2c5a-5596-4b6c-c006a02233f9,,,Obstetric ultrasound,"Échographie obstétricale ",Obstetric ultrasound,"Échographie obstétricale ",,,ConvSet,N/A,,,Foetus Details Set; Placenta; Amniotic liquid; Pregnancy duration; Other observations; Identified complications; Complications;,,,,,,,,
+e8ef8004-2d43-43d9-be1d-f1f7f894f99c,,,Observations,Observations,Observations,Observations,,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv]: Is this different from the 'Other observations' concept above?

File: configuration/concepts/echographie.csv [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+53b255ca-66f2-702b-1c4f-5f4baba006fe,,,Other ultrasound,Autre échographie,Other ultrasound,Autre échographie,,,ConvSet,N/A,,,Observations,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv]: Not sure what this one represents

File: configuration/concepts/misc.csv [hunk 9]
```diff
@@ -22,4 +22,12 @@ f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,See comments,Commentaires,Co
...
 75e366a1-c441-4c94-9c53-312c9b8cc550,,CIEL:1175,Not Applicable,Not Applicable,Non applicable,Non applicable,,,Misc,N/A,,,,,,,,,,,
 6b32bbbe-8a7c-4160-99ff-0dd6b255f458,,CIEL:1067,Unknown,Unknown,Inconnu,Inconnu,,,Misc,N/A,,,,,,,,,,,
-e605731b-2e81-41a9-8446-2ed442c339e2,,CIEL:1640,Date,Date,Date,Date,,,Misc,Date,,,,,,,,,,,
\ No newline at end of file
+036f30c9-ccff-4752-8206-722724cbbf49,,CIEL:1115,Normal,Normal,Ordinaire,Ordinaire,,,Misc,N/A,,,,,,,,,,,
+7c4b5dff-ade6-4fab-9452-b3be4bcba89e,,CIEL:1116,Abnormal,Abnormal,Anormal,Anormal,,,Misc,N/A,,,,,,,,,,,
+f7139b53-6e10-4096-8e87-ea133fc8fc46,,CIEL:164371,Not Seen,Not Seen,Non vu,Non vu,,,Misc,N/A,,,,,,,,,,,
+48be742e-e22e-4bd9-bb2b-0d4840c60aeb,,,Boy,Boy,Garçon,Garçon,,,Misc,N/A,,,,,,,,,,,
+7adb833e-9e48-4b76-aa8f-24daa3acb810,,,Girl,Girl,Fille,Fille,,,Misc,N/A,,,,,,,,,,,
+5c75e55f-845c-4df6-a6da-b45475a4a215,,,Weight Percentile,Weight Percentile,Centile de poids,Centile de poids,,,Misc,Numeric,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv]: In french, that'll be 'Percentile'

**Review by rbuisson — CHANGES_REQUESTED** [review]
See comments

File: configuration/concepts/echographie.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: Done

File: configuration/concepts/echographie.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: Done

File: configuration/concepts/echographie.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: Done

File: configuration/concepts/echographie.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: Done

File: configuration/concepts/echographie.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv:49]: The other we have is more less an answer, so will keep `Other Observations` for capturing others

File: configuration/concepts/echographie.csv (hunk 8 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: Removed

File: configuration/concepts/misc.csv (hunk 9 above)
Comment (reagan-meant) [reply configuration/concepts/misc.csv]: Fixed

File: configuration/concepts/echographie.csv [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+a999e659-d882-425e-b3db-a2d039d15ac6,,,Other Ectopic Pregnancy,Autre grossesse extra-utérine,Other Ectopic Pregnancy,Autre grossesse extra-utérine,,,Diagnosis,N/A,,,,,,,,,,,
+b4b1d099-1932-4eda-bc72-7ba69b471b3a,,,Intrauterine Pregnancy,Grossesse intra-utérine,Intrauterine Pregnancy,Grossesse intra-utérine,,,Diagnosis,N/A,,,,,,,,,,,
+309fba9e-7936-42e6-a7b6-d05830f5a432,,,Location of pregnancy,Localisation de la grossesse,Location of pregnancy,Localisation de la grossesse,,,Misc,Coded,,Intrauterine Pregnancy; Abdominal Pregnancy; Tubal Pregnancy; Ovarian Pregnancy; Other Ectopic Pregnancy,,,,,,,,,
+05e696ee-9330-483d-8419-1c587a5dd84e,,,Cervical Length,Longueur cervicale,Cervical Length,Longueur cervicale,,,Misc,Numeric,,,,TRUE,cm,,,,,,
+6422a83f-acc6-4953-8239-f18a0b6d90a4,,,Estimated date of delivery,Date prévue d'accouchement,Estimated date of delivery,Date prévue d'accouchement,,,Misc,Date,,,,,,,,,,,
+9e071d37-ee01-4785-a756-7e4059ffe736,,CIEL:1438,Gestational Age,Âge gestationnel par ultrasons,Gestational Age,L'âge gestationnel,,,ConvSet,N/A,,,Number of Weeks; Number of Days,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/echographie.csv:44]: remove the word ultrasound from the fr locale

**Pull request merged**